scap/src/capturer/engine/mac/mod.rs

```diff
@@ -154,41 +154,41 @@ pub fn create_capturer(options: &Options, tx: mpsc::Sender<Frame>) -> SCStream {
     stream
 }
 
-pub fn ycbcr_to_rgb(
-    y_data: &[u8],
-    cbcr_data: &[u8],
-    width: usize,
-    height: usize,
-    stride: usize,
-) -> Vec<u8> {
-    let mut rgb_data = Vec::with_capacity(width * height * 3);
-    let row = width + stride;
-
-    for h in 0..height {
-        for w in 0..width {
-            let y_idx = h * row + w;
-            let uv_idx = (h / 2) * row + w - w % 2;
-
-            // let y = y_data[y_idx] as f32;
-            // let cb = cbcr_data[uv_idx] as f32 - 128.0;
-            // let cr = cbcr_data[uv_idx + 1] as f32 - 128.0;
-
-            // NOTE: The following values adjust for contrast and range
-            let y = (y_data[y_idx] as f32 - 16.0) * (255.0 / (235.0 - 16.0));
-            let cb = (cbcr_data[uv_idx] as f32 - 16.0) * (255.0 / (240.0 - 16.0)) - 128.0;
-            let cr = (cbcr_data[uv_idx + 1] as f32 - 16.0) * (255.0 / (240.0 - 16.0)) - 128.0;
-
-            let r = (y + 1.402 * cr).max(0.0).min(255.0) as u8;
-            let g = (y - 0.344136 * cb - 0.714136 * cr).max(0.0).min(255.0) as u8;
-            let b = (y + 1.772 * cb).max(0.0).min(255.0) as u8;
-
-            rgb_data.push(r);
-            rgb_data.push(g);
-            rgb_data.push(b);
-        }
-    }
-    rgb_data
-}
+// pub fn ycbcr_to_rgb(
```

Comment:
Nit: We can remove this function if we don't need it. It'll anyway be available in the git history if we need it again in the future.